docs/source/upgrade-guide/index.md

```diff
@@ -47,6 +47,10 @@ Volto 17 now uses Webpack 5.
 If you customized `razzle.config.js` for your project to change Webpack configuration
 or use Webpack plugins, you might need to make adjustments.
 
+### Razzle upgraded to version `4.2.18`
+
+Razzle has been upgraded to version `4.2.18`. It is recommended that you update your projects dependency on Razzle to this version in order to avoid duplication.
```

Comment:
```suggestion
Razzle has been upgraded to version `4.2.18`.
It is recommended that you update your project's dependency on Razzle to this version in order to avoid duplication.
```